apps/langchain/src/react_agent/graph.ts

```
@@ -0,0 +1,76 @@
+  ]);
+
+  // We return a list, because this will get added to the existing list
+  return { messages: [response] };
+}
+
+// Define the function that determines whether to continue or not
+function routeModelOutput(state: typeof MessagesAnnotation.State): string {
+  const messages = state.messages;
+  const lastMessage = messages[messages.length - 1];
+  // If the LLM is invoking tools, route there.
+  if ((lastMessage as AIMessage)?.tool_calls?.length || 0 > 0) {
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)